priv/repo/migrations/20201129084435_create_reports_search_results_table.exs

```diff
@@ -0,0 +1,12 @@
+defmodule GSearcher.Repo.Migrations.CreateReportsSearchResultsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:reports_search_results) do
+      add :report_id, references(:reports)
+      add :search_result_id, references(:search_results)
```

Comment:
Not sure but should we add the index for report_id and search_result_id 🤔

Reply:
Yeah, you're right. I'll add that 👍

Reply:
Not sure do we need to add constraint on the `ReportSearchResult ` schema module as well. Just a small improvement though. 🆗